### ISSUE TITLE
Hazardous alcohol use

### ISSUE THREAD

**Issue by janful** [obophenotype/human-phenotype-ontology#10165]
**Preferred term label:**
Hazardous alcohol use

**Synonyms**
risky alcohol consumption, drink driving, drunk driving

**Definition (free text, please give PubMed ID)**
Hazardous drinking is defined as a quantity or pattern of alcohol consumption that places individuals at risk for adverse health events. This could include alcohol use despite potentially hazardous or risky consequences, such as operating heavy machinery or drunk driving.
PMID: 10448769 for definitions of Heavy, Hazardous, Harmful Alcohol use; https://jamanetwork.com/journals/jamainternalmedicine/fullarticle/1105638

**Parent term (use hpo.jax.org/app)**
[HP:5200329](https://hpo.jax.org/app/browse/term/HP:5200329) Problematic alcohol consumption

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000028
- label: Hazardous alcohol use
- parents: [Problematic alcohol consumption (HP:5200329)](https://hpo.jax.org/app/browse/term/HP:5200329)
- pmid: PMID:10448769
- def: Hazardous drinking is defined as a quantity or pattern of alcohol consumption that places individuals at risk for adverse health events. This could include alcohol use despite potentially hazardous or risky consequences, such as operating heavy machinery or drunk driving.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10165
- synonyms:
